static/app/components/group/sidebar.tsx

```diff
@@ -312,11 +296,30 @@ class BaseGroupSidebar extends Component<Props, State> {
 
         {this.renderPluginIssue()}
 
-        {(!organization.features.includes('issue-details-tag-improvements') ||
-          !(
-            isMobilePlatform(project.platform) &&
-            (project.platform === 'react-native' || organization.id === '1')
-          )) && (
+        {organization.features.includes('issue-details-tag-improvements') ? (
+          <TagFacets
+            environments={environments}
+            groupId={group.id}
+            tagKeys={
+              isMobilePlatform(project?.platform)
+                ? MOBILE_TAGS
+                : frontend.some(val => val === project?.platform)
+                ? FRONTEND_TAGS
+                : backend.some(val => val === project?.platform)
+                ? BACKEND_TAGS
+                : DEFAULT_TAGS
+            }
+            title={
+              <div>
+                {t('Tag Summary')} <FeatureBadge type="beta" />
+              </div>
+            }
+            event={event}
+            tagFormatter={TAGS_FORMATTER}
+            style={tagFacetsStyle}
```

Comment:
We should drop this style prop because I'm seeing different tag summary styled on different orgs

Reply:
good catch!